plugins/src/main/kotlin/extension/DependencyHandleScope.kt

```diff
@@ -114,6 +114,7 @@ fun DependencyHandlerScope.allServicesImpl() {
     // For analytics configuration, either use noop, or use the impl, with at least one analyticsproviders implementation
     // implementation(project(":services:analytics:noop"))
     implementation(project(":services:analytics:impl"))
+    implementation(project(":services:analytics:compose"))
```

Comment:
Wanting to check how it behaves when not including `:services:analytics:impl`. It seems to be OK.

I may want to revisit how we include or not modules by creating a configuration file, but I''l do it in antoher PR.